esp-hal/src/uart.rs

```diff
@@ -463,6 +468,20 @@ impl Config {
         self
     }
 
+    /// Sets duration between transfers in bits. The duration of time would
+    /// be based on your configured baud rate. If you are expecting bytes
+    /// written to be sent immediately, set this to 0. Default value is 256,
+    /// maximum value is 1023.
+    pub fn tx_idle_num(mut self, tx_idle_num: u16) -> Self {
```

Comment:
Please remove this, the `BuilderLite` macro already defines a setter. If you need validation, you can do it in apply_config - which you already do.

Reply:
Handled in [67e8d73](https://github.com/esp-rs/esp-hal/pull/2859/commits/67e8d737217c5ce0671c9fa81a5f437b73d53ae7)